src/main/java/emissary/util/ObjectTracing.java

```diff
@@ -0,0 +1,27 @@
+package emissary.util;
+
+import emissary.core.IBaseDataObject;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ObjectTracing {
+
+    public ObjectTracing() {}
+
+    /**
+     * Given an IBDO, create a map with log entries we care about and return it
+     * 
+     * @param d the IBDO
+     * @return the map of log entries to add to the log
+     */
+    public Map<String, String> createTraceMessageMap(IBaseDataObject d) {
+
+        Map<String, String> jsonMap = new HashMap<>();
+
+        // add our fields
+        jsonMap.put("inputFileName", d.getFilename());
+        jsonMap.put("stage", "PickUpPlace");
```

Comment:
I agree using the place name probably isn't the best here. Using stage names like PickUp, DropOff might be more clear for us to understand exactly where in processing we are. Since right now we are only adding this logging for the "PickUp" stage, we can always modify or add to the names later if we would like.